addons/medical_engine/functions/fnc_getHitpointArmour.sqf

```diff
@@ -0,0 +1,73 @@
+#include "script_component.hpp"
+/*
+ * Author: Pterolatypus
+ * Checks a unit's equipment to calculate the total armour on a hitpoint
+ *
+ * Arguments:
+ * 0: Unit <OBJECT>
+ * 1: Hitpoint <STRING>
+ * 2: Disable Caching <BOOLEAN> (Default: false)
+ *
+ * Return Value:
+ * Total armour for the given hitpoint <ARRAY>
+ * 0: Armour <NUMBER>
+ * 1: ExplosionShielding <NUMBER>
+ *
+ * Example:
+ * [player, "HitChest"] call ace_medical_engine_fnc_getHitpointArmour
+ *
+ * Public: No
+ */
+
+params ["_unit","_hitpoint",["_noCache",false]];
```

Comment:
What's the point of noCache? Seems like bloat to me. If you want to clear the cache for debugging, delete the namespaces.